advanced_html_css/animation/keyframes.md

```diff
@@ -132,4 +132,4 @@ This section contains questions for you to check your understanding of this less
 
 ### Additional resources
 
-This section contains helpful links to other content. It isn't required, so consider it supplemental if you need to dive deeper into something.
+[Video series on CSS Animations from DevTips](https://www.youtube.com/playlist?list=PLqGj3iMvMa4LvJ8VctoXnPI0dtE40wfid).
```

Comment:
This full playlist might be a bit much for this lesson. It covers various different animations subjects besides keyframes. Some of our learners aim to complete all additional resources on top of the assignments, I'd be reluctant to give them 3.5 hours of videos to go through here.

Would it be worth just linking to the keyframes video in that playlist instead?

Reply:
I considered supplemental material as an optional way to revise the lesson's material, I personally bookmark links the lessons which have them; as you likely have the data about the learning preference of the learners, your decision would clearly be more appropriate :) 
I attached the link to the entire series considering someone might not notice that the video is a part of really informative video series on animations